tests/unit-tests/Runners/EchoFakeRunnerTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sergey
+ * Date: 30.01.17
+ * Time: 12:30
+ */
+
+namespace AdamBrett\ShellWrapper\Tests\Runners;
+
+
+use AdamBrett\ShellWrapper\Command;
+use AdamBrett\ShellWrapper\Runners\EchoFakeRunner;
+
+class EchoFakeRunnerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testRun()
+    {
+        $runner = new EchoFakeRunner();
+        $runner->run(new Command('ls'));
```

Comment:
could you wrap this in `ob_start` and `ob_get_clean` to test that the output matches the expected value?